packages/ledger/conflux.js

```diff
@@ -131,8 +147,9 @@ export default class Conflux {
       for (const index of indexArray) {
         const hdPath = `${HDPATH.CONFLUX}${index}`
         const {address} = await this.getAddress(hdPath)
+        const {hexAddress} = decode(address)
         addressArr.push({
-          address,
+          address: hexAddress,
```

Comment:
will this broke the frontend code at https://github.com/Conflux-Chain/helios/blob/b1eda8c8712e394a2e999b9283c1db8b5a6ad982/packages/popup/src/pages/ImportHwAccount/index.js#L109

Reply:
nope, the frontend code requires the hex-format address